styles.css

```diff
@@ -66,4 +74,167 @@ body {
     list-style: none;
     flex: 1;
   }
+
+  .projects_header {
+    position: absolute;
+    width: 327px;
+    height: 200px;
+    top: 100px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    color: #172b4d;
+    order: 0;
+    flex-grow: 0;
+    margin: 24px;
+  }
+
+  .center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  #down_icon {
+    margin-top: 40px;
+  }
+
+  .grid_container {
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: auto auto auto auto auto auto;
+    padding-top: 75px;
+    justify-content: center;
+  }
+
+  .flex_container_primary {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #d0d9d4;
+    border-radius: 15px;
+    margin-bottom: 150px;
+    height: 474px;
+  }
+
+  .flex_container_secondary {
+    padding: 50px;
+    gap: 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .flex_container_primary h3 {
+    text-align: center;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    color: #3a4a42;
+  }
+
+  .flex_container_primary ul {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    justify-content: center;
+  }
+
+  .flex_container_primary li {
+    flex: none;
+    list-style-type: none;
+    padding: 8px 12px;
+    margin: 0 8px;
+    border-radius: 4px;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    color: #3a4a42;
+    background: #ebf0ee;
+  }
+
+  a.web_button {
```

Comment:
- [ ] All your buttons (the "see project" and "get my resume") should have their actions, such as hover, styled. You can check the styles of the buttons in the Figma design:

![image](https://user-images.githubusercontent.com/66967274/143683373-8cb28ba8-bf31-432f-8b7a-c81eead56591.png)